internal/pkg/cli/flag.go

```diff
@@ -202,7 +202,7 @@ const (
 	resourceTagsFlagDescription = `Optional. Labels with a key and value separated by commas.
 Allows you to categorize resources.`
 	stackOutputDirFlagDescription  = "Optional. Writes the stack template and template configuration to a directory."
-	uploadResourcesFlagDescription = "Optional. Whether to upload dependency resources (e.g., an image that needs to be built and pushed), so that the output of `copilot svc package` can be directly used for deployment."
+	uploadResourcesFlagDescription = "Optional. Whether to upload dependency resources (e.g., an image that needs to be built and pushed), so that the output can be directly used for deployment."
```

Comment:
Should we rename the flag to `uploadAssets`? to use the same terminology as CDK: https://docs.aws.amazon.com/cdk/v2/guide/assets.html

```suggestion
	uploadResourcesFlagDescription = `Optional. Whether to upload assets (container images, Lambda functions).
Uploaded asset locations are filled in the template configuration.`
```